client/components/domains/domain-product-price/index.jsx

```diff
@@ -15,12 +18,17 @@ const DomainProductPrice = React.createClass( {
 		requiresPlan: React.PropTypes.bool
 	},
 	renderFreeWithPlan() {
+		const product_price = ! domainsWithPlansOnlyTestEnabled &&
+							<span
+								lassName="domain-product-price__price">{ this.translate( '%(cost)s {{small}}/year{{/small}}', {
+									args: { cost: this.props.price },
+									components: { small: <small /> }
+								} ) }</span>;
+
 		return (
-			<div className="domain-product-price is-free-domain">
-				<span className="domain-product-price__price">{ this.translate( '%(cost)s {{small}}/year{{/small}}', {
-					args: { cost: this.props.price },
-					components: { small: <small /> }
-				} ) }</span>
+			<div
+				className={ 'domain-product-price is-free-domain' + ( domainsWithPlansOnlyTestEnabled && ' no-price' ) }>
```

Comment:
The common (and probably recommended) way of conditionally including class names is via... `classnames` ;) There are many examples in our code and then there's (official documentation](https://github.com/JedWatson/classnames#usage). In this case it'd be something like this:
```
classNames( 'domain-product-price', 'is-free-domain', { 'no-price': domainsWithPlansOnlyTestEnabled } );
```